pkgs/by-name/gt/gtultra/package.nix

```diff
@@ -0,0 +1,115 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  fetchpatch,
+  alsa-lib,
+  copyDesktopItems,
+  imagemagick,
+  makeDesktopItem,
+  pkg-config,
+  SDL2,
+  darwin,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "gtultra";
+  version = "1.5.5";
+
+  src = fetchFromGitHub {
+    owner = "jpage8580";
+    repo = "GTUltra";
+    rev = "refs/tags/v${finalAttrs.version}";
+    hash = "sha256-hewj7qrPx4V6lSjbiCIB8yTAmfXxm5HEzB+wftkPt4E=";
+  };
+
+  patches = [
+    # https://github.com/jpage8580/GTUltra/pull/22
+    ./fix-format.patch
+    # https://github.com/jpage8580/GTUltra/pull/73
+    ./fix-buffer-overflow.patch
+
+    # Fix missing inclusion of Darwin target conditionals
+    (fetchpatch {
+      url = "https://github.com/thestk/rtmidi/commit/9d863beb28f03ec53f3e4c22cc0d3c34a1e1789b.patch";
+      stripLen = 1;
+      extraPrefix = "src/";
+      hash = "sha256-t3xCJxZk2Rk0XDZ4s8Yt8KOHsUNfeucr0aF+xmIW8OY=";
+    })
+  ];
+
+  postPatch = ''
+    # remove prebuilts
+    rm linux/* mac/* win32/*
+    rm -r src/datafiles
+
+    # Remove unconditional stripping
+    substituteInPlace src/makefile.common \
+      --replace-fail 'strip $@' '#strip $@'
+
+    substituteInPlace src/makefile.mac \
+      --replace-fail "\`sdl2-config --cflags'" '`sdl2-config --cflags`'
+  '';
+
+  nativeBuildInputs = [
+    imagemagick
+    copyDesktopItems
+    pkg-config
+  ];
+
+  buildInputs =
+    [ SDL2 ]
+    ++ lib.optional stdenv.isLinux alsa-lib
+    ++ lib.optionals stdenv.hostPlatform.isDarwin [
+      darwin.apple_sdk.frameworks.CoreAudio
+      darwin.apple_sdk.frameworks.CoreFoundation
+      darwin.apple_sdk.frameworks.CoreMIDI
+    ];
+
+  # Uses sdl2-config
+  strictDeps = false;
```

Comment:
It should be possible to just replace `sdl2-config` in all the Makefiles that use it with `pkg-config sdl2` instead of opting out of this, but not a blocker.